extra/utils/permision.js

```diff
@@ -0,0 +1,70 @@
+let permissions= {
+    'getUsers': {
+        'all': ['head-trainer'],
+        'read': ['trainee', 'trainer'],
+        'write': ['trainer'],
+        'delete': [],
+    }
+}
+
+function hasPermission(moduleName,role,permissionType)
+{
+   
+    if(permissions[moduleName]!== undefined)
+    {
+        console.log("Module Name ="+moduleName)
+        if(permissions[moduleName][permissionType]!==undefined)
+        {
+            console.log("Permission Type ="+permissionType)
+            if(permissions[moduleName][permissionType]!=='all')
+            {
+                if(permissions[moduleName]['all'].indexOf(role)!==-1)
+                {
+                    console.log("Role ="+role)
+                    return true;
+                }
+            }
+            if(permissions[moduleName][permissionType].length>0)
```

Comment:
No need to check length if array is empty indexOf will return -1. So no need for this check.